linux/fredonia_linux/manifests/services.pp

```diff
@@ -1,3 +1,73 @@
 class fredonia_linux::services () {
-  #Filler
+  #services to manage 
+  #optional(if time allows): add notifications
+  # to update log when services are run & track
+  # for debugging 
+  #Samba running and enabled
+  package { 'samba':
+    ensure   => 'installed',
+    provider => 'apt',
+    before   => Service['samba'],
+  }
+  service { 'samba':
+    ensure => running,
+    start  => '/usr/sbin/smbd service start',
+    stop   => '/usr/sbin/smbd service stop',
```

Comment:
In regards to the start/stop.

Since samba should add a daemon by default, it would be better to leverage the systemd scripts laid down by the service (you should be able to leave this as ensure => running without specifying a command).

(systemctl status smb)